packages/edit-site/src/components/sidebar-button/index.js

```diff
@@ -11,8 +11,7 @@ import { Button } from '@wordpress/components';
 export default function SidebarButton( props ) {
 	return (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
This can be seen when you visit, site-editor --> and check for icon button in sidebar panel.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="Editor-sidebar-button-before" src="https://github.com/user-attachments/assets/1708c69c-7c06-4fa5-b609-8ffafcfc16ba"> | <img width="419" alt="Editor-sidebar-button-after" src="https://github.com/user-attachments/assets/71c7224e-b4a0-4168-baf0-5aa5850d18e2"> |